### DISCUSSION TITLE
PNGImage not found error

### DISCUSSION THREAD

**Discussion by bzwirs** [AngusJohnson/Image32#28] [General]
Have just started to try Image32 and I get the following error when compiling my project.  Using Delphi 11.3 on multi device FMX project.

[dcc32 Fatal Error] Img32.Fmt.PNG.pas(20): F2613 Unit 'PngImage' not found.

Can anyone please point me in the right direction to solve this.

Thanks in advance.

Bill

**Comment by AngusJohnson** [comment]
`Vcl.Imaging.pngimage.pas` in `$(BDS)\source\vcl`

Edit: But in FMX you can't use `Img32.Fmt.PNG`, use `Img32.Fmt.FMX` instead.
